docs/development.md

```diff
@@ -92,44 +95,49 @@ following chapters in a certain order:
     - spin up a bare redis container
 
       ```shell-session
-      docker run -d -p 6379:6379 --restart unless-stopped redis:latest
+      $ docker run -d -p 6379:6379 --restart unless-stopped redis:latest
       ```
 
-7.  Install the python dependencies by performing in the src/ directory.
+7.  Install the python dependencies by performing in the root directory:
 
     ```shell-session
-    pipenv install --dev
+    $ cd /path/to/paperless
+    $ pipenv install --dev
     ```
 
-!!! note
+    !!! note
 
-    Make sure you're using python 3.10.x or lower. Otherwise you might
-    get issues with building dependencies. You can use
-    [pyenv](https://github.com/pyenv/pyenv) to install a specific
-    python version.
+        Using a virtual environment is highly recommended.
```

Comment:
`pipenv` creates and manages a virtual environment already, so a user doesn't need to do this manually.

Reply:
It will only create a virtual environment if you tell it to do so, if I understand it correctly. So I thought this would be helpful.

Reply:
An `install` or `sync` creates a virtual environment and installs dependencies to it, so this step already creates one.  Ad advanced user could generate `requirements.txt` and create their own venv, but advanced users can sort out their own problems :)

Reply:
Thank you for your continuous input - being a beginner I might be a little bit too verbose at certain points. In the latest commit you'll find a similar wording around line 80 in `development.md`. So I'll just remove this part then?

Reply:
The current wording looks good, with the pointer to use pipenv instead of manually creating one.